attestation-service/src/bin/restful-as.rs

```diff
@@ -96,7 +97,9 @@ async fn main() -> Result<(), RestfulError> {
 
     let attestation_service = web::Data::new(Arc::new(RwLock::new(attestation_service)));
     let server = HttpServer::new(move || {
+        let cors = Cors::permissive();
```

Comment:
The official document says `permissive()`: All origins, methods, request headers and exposed headers allowed. (**Not recommended** for production use.)

It would be a good choice to add a parameter for restful named `--allowed-origin` or something similar to set manually when launching. If this parameter is not given, we fully forbid oras.

The reference of parameter definition is [here](https://github.com/confidential-containers/trustee/blob/main/attestation-service/src/bin/restful-as.rs#L37-L40)

Reply:
agree with @Xynnn007. `Cors::permissive()` is not a proper fix.

Reply:
Done